packages/config/config/devices/0x031e/lzw45.json

```diff
@@ -0,0 +1,1169 @@
+		"18": {
+			"label": "Periodic Power & Energy Reports",
+			"description": "Time period between consecutive power & energy reports being sent (in seconds). The timer is reset after each report is sent. Range: 0, 30 - 32767 Default: 3600",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 3600,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"19": {
+			"label": "Energy Reports",
+			"description": "The energy level change that will result in a new energy report being sent (in kWh)\n\t  0 = 0.01kWh, 10 = 0.1kWh, 100 = 1kWh. Range:0-127 Default: 10",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21[0xff]": {
+			"label": "Quick Strip Effect: Hue Color Wheel / Color Temp",
+			"description": "If 'Quick Strip Effect: Color Type' is set to 'Hue Color Wheel', this is the hue color wheel value.  To calculate value, value = hue_color_wheel_value / 360 * 255.  If 'Quick Strip Effect: Color Type' is set to 'Color Temp', this is the color temp value.  To calculate value, value = (color_temp_value - 2700) / (6500 - 2700) * 255. Range: 0-255  Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21[0x7f00]": {
+			"label": "Quick Strip Effect Intensity",
+			"description": "Intensity of the LED strip.  Course: 0 - 10, where 10 = 100%.  Fine: 0-99, where 99 = 100%.  Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21[0x8000]": {
+			"label": "Quick Strip Effect Intensity Scale",
+			"description": "Changes the scale of the intensity.  0 = Course (10 x Intensity), 1 = Fine (1 x Intensity).  Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Course",
+					"value": 0
+				},
+				{
+					"label": "Fine",
+					"value": 1
+				}
+			]
+		},
+		"21[0xff0000]": {
+			"label": "Quick Strip Effect: Duration",
+			"description": "The light LED strip effect duration. 1 to 60 = seconds, 61 to 120 minutes, 121 - 254 = hours, 255 = Indefinitely.  Default: 0",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21[0x3f000000]": {
+			"label": "Quick Strip Effect: Effect",
+			"description": "The light LED strip effect. 0 = Off, 1 = Solid, 2 = Chase, 3 = Fast Blink, 4 = Slow Blink, 5 = Fast Fade, 6 = Slow Fade.  Default: 0",
```

Comment:
remove

Reply:
remove what?  The whole description or the values?

Reply:
I'm going to assume you mean the 0 = off etc.